websites/P/Pulse/presence.ts

```diff
@@ -26,30 +26,12 @@ newStats();
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "logo",
-    startTimestamp: browsingStamp
+    
+    
   };
-  if (document.location.pathname.startsWith("/lite")) {
-    play = document.querySelector(
-      "body > div > div > div > div > div > div > i"
-    );
-    play = play.className;
-    switch (play) {
-      case "fa fa-play":
-        // Music Paused
-        presenceData.smallImageKey = "pause";
         presenceData.state = (strack || "Loading...") + (" - " + sartist);
         presenceData.details =
           (sdj || "Loading...") + (slisteners || "Loading...") + " Listeners";
```

Comment:
actually you can move this to presenceData object since you are not changing it.